python/docs/_templates/announcement.html

```diff
@@ -0,0 +1,3 @@
+<div class="announcement">
```

Comment:
Could maybe have this use a config in the quarto.yml, and only show up if that config is set? Alternatively.. should we just drop the announcement at this point?